tests/test_ijit/test_ijit_without_numba.py

```diff
@@ -19,5 +19,25 @@ def _simple_func(a, b):
         assert len(w) >= 1
         assert isinstance(_simple_func, types.FunctionType)
 
+@mock.patch.dict(sys.modules, {"numba": None})
```

Comment:
`test_function_given` and `test_no_function_given` can be combined into a single test.